content-services/content-sources-backend
# Fixes 1580: Add API to list and fetch tasks

## Summary

Adds API to list tasks:

`GET /api/content-sources/v1.0/tasks/`

and to fetch a task:

`GET /api/content-sources/v1.0/tasks/:uuid`

It defines a GORM model that is shared between the queue and DAO. The DAO only uses GORM to read in tasks.

## Testing steps

Start an introspection task

Make a GET request to `/api/content-sources/v1.0/tasks/` to see all tasks

Make a GET request to `GET /api/content-sources/v1.0/tasks/:uuid` with the task UUID to the details of the specific task
